cmd/lncli/commands.go

```diff
@@ -680,25 +672,27 @@ func closeChannel(ctx *cli.Context) error {
 		SatPerByte:   ctx.Int64("sat_per_byte"),
 	}
 
+	args := ctx.Args()
+
 	switch {
 	case ctx.IsSet("funding_txid"):
-		txid = ctx.String("funding_txid")
+		req.ChannelPoint.FundingTxid = &lnrpc.ChannelPoint_FundingTxidStr{
```

Comment:
is this change needed?

Reply:
Not really, I just wanted to match how the `output_index` is parsed below.